WinverUWP/Strings/es-ES/Resources.resw

```diff
@@ -0,0 +1,162 @@
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="AboutWindows.Text" xml:space="preserve">
+    <value>Sobre Windows</value>
+  </data>
+  <data name="Copied" xml:space="preserve">
+    <value>Copiado!</value>
+  </data>
+  <data name="CopyButton.Content" xml:space="preserve">
+    <value>Copiar</value>
+  </data>
+  <data name="Edition.Text" xml:space="preserve">
+    <value>Edición</value>
+  </data>
+  <data name="Experience.Text" xml:space="preserve">
+    <value>Experience</value>
+  </data>
+  <data name="ExperiencePack" xml:space="preserve">
+    <value>Windows Feature Experience Pack</value>
+  </data>
+  <data name="InstalledOn.Text" xml:space="preserve">
+    <value>Instalado en</value>
+  </data>
+  <data name="LegalLabel.Text" xml:space="preserve">
+    <value>Legal</value>
+  </data>
+  <data name="LicenseTo.Text" xml:space="preserve">
+    <value>Este producto está licenciado bajo la Licencia de Software de Microsoft a:</value>
+  </data>
+  <data name="OKButton.Content" xml:space="preserve">
+    <value>Aceptar</value>
```

Comment:
From the text, it looks like "Accept" which I don't think is quite related to "OK".